mLRS/Common/hal/esp/tx-hal-radiomaster-bandit-micro-900-esp32.h

```diff
@@ -10,7 +10,9 @@
 
 /*
   Flashing ESP8285:
-  - put Tx module into FLASH_ESP mode
+  - change ser dest to serial2
+  - make sure baudrate is 115200 (default)
```

Comment:
is this for the tx module, or the setting in Arduino?

I think in the tx module one does not have to use 115200. If right, and it's the ArduinoIDE, I would list it at the end with "Baudrate: 115200" (to make it more clear its's in the IDE

Reply:
Have changed to 'Upload Speed: 115200' to align with the menu item in the Arduino IDE.

PS, news to me that the Tx module doesn't need this setting to any specific value?

Reply:
the com port (when via UART) is always at 115200
doesn't the esp do autobauding when in flash mode?

the baudrate of the serial/serial2 and that coded into the wireless bridge code need to match, true, but I think that's not for flashing ... but better not trust my word but check LOL

Reply:
> but I think that's not for flashing ... but better not trust my word but check LOL

This seems to not be the case.  If I set to tx ser baud to 9600 then I can no longer flash.

Reply:
hm ... maybe 9600 is to slow for the IDE runnig at 115200 to get the timing right ... what woukd happen with 57600, 230400?
it would be sad if those who want to use 230400 on the serial essentially couldn't anymore ...

Reply:
For me, the tx ser baudrate needs to align with the upload speed in the IDE.  That being said the ESP8266 offers interesting options - maybe we allow 921600 baud?

![image](https://github.com/user-attachments/assets/83673aa2-6f39-48a3-8d53-885b65e93cc9)

Reply:
> For me, the tx ser baudrate needs to align with the upload speed in the IDE to upload

I'm happy to accept this as a fact :)

so we should add a lin like you had before (maybe just making it clearer it's in the tx)